processors/github/github_processor_phase1_fixed.py

```
@@ -0,0 +1,747 @@
+            self.db.create_collection(collection_name)
+            logger.info(f"Created collection: {collection_name}")
+        
+        # Setup indexes
+        repo_coll = self.db.collection(collection_name)
+        try:
+            repo_coll.add_persistent_index(
+                fields=['file_path'],
+                sparse=False,
+                name='idx_file_path'
+            )
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)